pkg/kv/sender.go

```diff
@@ -231,6 +231,11 @@ type TxnSender interface {
 	// field on TxnMeta.
 	ProvisionalCommitTimestamp() hlc.Timestamp
 
+	// GlobalUncertaintyLimit returns the transaction's global uncertainty
+	// limit, which is the maximum clock drift observed across. This is used
+	// for dealing with clock time vs MVCC time for certain hueristics.
+	GlobalUncertaintyLimit() hlc.Timestamp
```

Comment:
Something seems off about the need to expose this. Also, I was envisioning a duration. This deserves commentary if we need to expose it.